tests/cudnn_fusion_test.py

```diff
@@ -26,8 +27,9 @@
 class CudnnFusionTest(jtu.JaxTestCase):
   def setUp(self):
     if (not jtu.test_device_matches(["cuda"]) or
-        not jtu.is_cuda_compute_capability_at_least("9.0")):
-      self.skipTest("Only works on >= sm90 GPUs")
+        not jtu.is_cuda_compute_capability_at_least("8.0") or
+        cuda_versions.cudnn_get_version() < 9110):
```

Comment:
I figured out the problem in this PR: the version number should be 90110.

I guess I'll change that API to return a tuple to avoid this kind of problem.

Reply:
Thank you! Fixed.